src/components/Header/Header.jsx

```diff
@@ -0,0 +1,135 @@
+import { Fragment } from 'react'
+import { Popover, Transition } from '@headlessui/react'
+import {
+  MenuIcon,
+  XIcon,
+} from '@heroicons/react/outline'
+
```

Comment:
Please add semicolons at the end of each expression or install **Prettier** extension to VS Code. That will automatically add the semicolons for you